e2e/global.setup.ts

```
@@ -0,0 +1,10 @@
+import { clerkSetup } from '@clerk/testing/playwright';
+import { test as setup } from '@playwright/test';
+
+// Setup must be run serially, this is necessary if Playwright is configured to run fully parallel:
+// https://playwright.dev/docs/test-parallel
+setup.describe.configure({ mode: 'serial' });
+
+setup('global setup', async ({}) => {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)